### ISSUE TITLE
Mixtec forms not updated?

### ISSUE THREAD

**Issue by GuillemBelmar** [gtroiani/yucunani_catchgame#17]
Jeremías and I corrected the mixtec forms today, and I re-downloaded the game. If I open the json file the forms are correct (aka what we did this morning), but they're not updated in the game.

**Comment by gtroiani** [comment]
@GuillemBelmar  I tested it with a new localization file and it worked for me. If you have not tried it yet, I suggest the following:

- check that the naming conventions for your file are correct (filename "locale_**.json" , where "**" needs to correspond to the two-letter code for the language that is the first pair in the JSON dictionary)
- check the authorization to write the Program folder when you paste the localization file 
- close and restart the game

If nothing of this works, please give me a description of the steps you followed so I can replicate the bug

**Comment by GuillemBelmar** [comment]
Ok, so... Remember I told you I modified the json files directly on github? When I go to code - datafiles and open the json files, they are updated with those changes. But I then download the game again and when I open the downloaded json files, they are not updated. They're the old ones. I added the P'urhepecha to the folder and that works, so it's not so much about the localization. It's that I changed it from github directly, but it doesn't seem to be updated when downloading.
